frontend/app/src/components/settings/data-security/backups/DatabaseBackups.vue

```diff
@@ -2,10 +2,10 @@
   <fragment>
     <data-table
       :value="selected"
-      :items="items"
+      :items="itemsWithIdentifier"
       sort-by="time"
       show-select
-      item-key="time"
+      item-key="identifier"
```

Comment:
keep in mind that if you have many backups this can cause issues with the table rendering. Maybe we can use the index if we only need this for selection or expansion.